ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp

```diff
@@ -131,9 +200,13 @@ void AirsimROSWrapper::create_ros_pubs_from_settings_json()
         set_nans_to_zeros_in_pose(*vehicle_setting);
 
         vehicle_name = curr_vehicle_name;
-        odom_pub = nh_.advertise<nav_msgs::Odometry>("odom", 10);
+        odom_pub = nh_.advertise<nav_msgs::Odometry>("for_testing/odom", 10);

```

Comment:
not in line with docs

Reply:
I can't find a place where odom is mentioned without the prefix

Reply:
the inconsistency was that some were `for_testing` and some `testing`. Thanks for fixing it!